src/pages/_error.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Rollbar from 'rollbar';
+
+function Error({ statusCode }) {
+  return (
+    <p>
+      {statusCode
+        ? `An error ${statusCode} occurred on server!`
+        : 'An error occurred on client'}
+    </p>
+  );
+}
+
+Error.getInitialProps = ({ req, res, err }) => {
```

Comment:
May be moot, but https://nextjs.org/docs/api-reference/data-fetching/getInitialProps suggest not using `getInitialProps` – is that relevant here?

Reply:
because we don't know if this page is server or static rendered until the time when the error is thrown we use getInitialProps to cover both scenarios.

Reply:
Makes sense.